app/src/test/java/com/futurice/freesound/feature/search/DefaultSearchDataModelTest.java

```diff
@@ -73,78 +72,76 @@ public void querySearch_emitsSingleUnit_whenQuerySearchSuccessful() {
         new Arrangement()
                 .withSearchResultsFor(QUERY, dummyResults());
 
-        TestSubscriber<Unit> ts = testSubscribe(defaultSearchDataModel.querySearch(QUERY));
+        TestSubscriber<Unit> ts = defaultSearchDataModel.querySearch(QUERY).test();
 
-        assertThat(ts).hasNoErrors()
-                      .hasReceivedValue(Unit.DEFAULT)
-                      .hasCompleted();
+        ts.assertValue(Unit.DEFAULT);
```

Comment:
Does the `TestSubscriber` replacement (`.test()`) now automatically assert those other conditions?